source/CMakeLists.txt

```diff
@@ -1,103 +1,58 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
The cmake version on debian 7 is '2.8.9'

Reply:
seen with you

Reply:
For future reading: no specific need for that upgrade.
But it's minor, we let it that way.